app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/SessionViewModel.java

```diff
@@ -117,6 +117,8 @@ private int decideColSpan(@NonNull Session session, int roomCount) {
             return 3;
         } else if (session.isBreak()) {
             return roomCount;
+        } else if (session.isDinner()) {
+            return 6;
```

Comment:
`roomCount` is always 6 on DroidKaigi2017. But it's calculated by sessions data.

And in my opinion, Session.Type should not have UI data like colSpan.
It's better to switch colspan in ViewModel.
What do you think?